cypress/e2e/upload/uploadKeywords.cy.ts

```diff
@@ -0,0 +1,47 @@
+import { SEARCH_RESULT_LIST_SELECTOR } from '../searchResults/selectors';
+
+import { UPLOAD_SELECTOR } from './selectors';
+
+describe('Uploads csv file with keywords', () => {
+  beforeEach(() => {
+    cy.task('resetDatabase');
+  });
+
+  describe('given there is valid file', () => {
```

Comment:
It can be shortened to:

```suggestion
  describe('given a valid CSV file', () => {
```